### ISSUE TITLE
Improve accessibility with keyboard/screen-reader

### ISSUE THREAD

**Issue by elwinschmitz** [rodekruis/helpful-information#265]
As a "user",  
I want to use my screen-reader, 
So that I can read all the information.  

## Details
The app/site should be accessible with a keyboard only anyway;
But there is a specific user that will be using the Q&A-instance of HIA specifically. 

## Requirements / In/Out of Scope
- Using;
  - OS: Windows
  - Browser: Edge (?)
  - Screen-reader: NVDA? Windows-built-in?
  - Display: high-contrast-mode?
  - Input-mode: Keyboard (laptop)

## Specific issues
- [x] ENV-label steals too much focus? Or is blocking the rest of the page? See: #262
- [x] Headings and lists should be defined in HTML accordingly
- [x] Focus management after navigation (Maybe: Use 1 `ion-content`, swap contents only. Or: multiple Page-components?)
